### PR TITLE
Add Test Cases

### PR DESCRIPTION
**Pull request by Jai2501**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/FavouriteCommandTest.java [hunk 1]
```diff
@@ -28,6 +29,8 @@
     private final Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), null);
     private final Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs(), null);
 
+    private Index targetIndex;
+
```
Comment (CheongYeeMing) [inline src/test/java/seedu/address/logic/commands/FavouriteCommandTest.java:33]: Did you add this by mistake? It does not seem to be used below.

**Review by CheongYeeMing — COMMENTED** [review]
I have settled and maxed the coverage for FavouriteCommandTest!

File: src/test/java/seedu/address/logic/commands/FavouriteCommandTest.java (hunk 1 above)
Comment (Jai2501) [reply src/test/java/seedu/address/logic/commands/FavouriteCommandTest.java]: Fixed it.

**Review by CheongYeeMing — APPROVED** [review]
LGTM! 👍

**Pull request merged**